.github/workflows/build-and-tag-maven.yml

```diff
@@ -80,6 +80,8 @@ jobs:
           MAVEN_CLI_OPTS: --show-version --no-transfer-progress --settings settings.xml -Dlogging.root.level=off -Dspring.main.banner-mode=off -Ddocker.skip
           MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
           MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+      - name: Echo Longest Tests run
+        run: find . -name TEST-*.xml -exec grep -h testcase {} \; | awk -F '"' '{printf("%s#%s() - %.3fms\n", $4, $2, $6); }' | sort -n -k 3 | tail -20
```

Comment:
are you aware of the nice actions that upload test summaries parsing xml files like:
https://github.com/marketplace/actions/publish-unit-test-results
https://github.com/marketplace/actions/test-reporter

they seems to also report test execution time e.g. https://github.com/dorny/test-reporter/blob/main/assets/provider-error-summary.png

don't see anything like the slowest test so in the end this approach is still valuable